server/bin/update-server-xml.py

```diff
@@ -137,42 +189,73 @@ def new_node(self):
 
     @property
     def attributes(self):
+        # We manually add the attributes below, so we don't want to return anything here.
+        return []
+
+    def _build_node(self):
         ciphers = ",".join([
-            "SSL_RSA_WITH_3DES_EDE_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA",
-            "TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA",
-            "TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA,TLS_ECDH_RSA_WITH_AES_128_CBC_SHA",
-            "TLS_ECDH_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA",
+            "SSL_RSA_WITH_3DES_EDE_CBC_SHA",
+            "TLS_RSA_WITH_AES_256_CBC_SHA",
+            "TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA",
+            "TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA",
+            "TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA",
+            "TLS_ECDH_RSA_WITH_AES_128_CBC_SHA",
+            "TLS_ECDH_RSA_WITH_AES_256_CBC_SHA",
+            "TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA",
         ])
 
-        # This is a list of tuples instead of a dict so we can preserve the attribute
-        # ordering.  OrderedDict didn't get added until 2.7.
-        return [
+        # Setup our node configuration
+        connector = libxml2.newNode("Connector")
+        self._add_attributes(connector, [
```

Comment:
We can probably actually use an OrderedDict now if we wanted.  Pretty much everything is 2.7+ these days.